pkg/webhook/webhook.go

```diff
@@ -175,3 +196,112 @@ func (c *Webhook) validateHandlerFunc(review *admission_v1beta1.AdmissionReview)
 		Allowed: true,
 	}
 }
+
+func (c *Webhook) validateRelease(request *admission_v1beta1.AdmissionRequest, release shipper.Release) error {
+	var err error
+	if request.Operation == kubeclient.Create {
+		err = c.shouldBlockRelease(release)
```

Comment:
Not sure I understand what's happening here: validateRequest is supposed to ensure a release object validity and return an error if something's up. In this line the error is assigned to an error returned by shouldBlockRelease, which implies on some boolean-returning idempotent call. The method name doesn't really reflect on any validation happening inside though.